plugins/org.jboss.tools.livereload.ui/plugin.xml

```diff
@@ -82,9 +82,9 @@
                  <iterate
                        ifEmpty="false"
                        operator="or">
-                    	<adapt
-							type="org.eclipse.wst.server.ui.IServerModule">
-						</adapt>
+                    	<instanceof
+							value="org.eclipse.wst.server.ui.IServerModule">
+						</instanceof>
```

Comment:
this change was applied in https://github.com/jbosstools/jbosstools-livereload/commit/67d2360fe1ed4478c8fbe200a6c00b9780520022. Please rebase/squash.